datarobot_batch_scoring/reader.py

```diff
@@ -379,6 +379,8 @@ def investigate_encoding_and_dialect(dataset, sep, ui, fast=False,
     else:
         opener = open
     with opener(dataset, 'rb') as dfile:
+        line_to_sniff = dfile.readline()
```

Comment:
@romankolpak I'm not sure you'll get enough interesting data here because this is just the header. The header will let us find the delimiter, but we probably won't find the interesting stuff like quoting until later on. Does that make sense?